api/src/messaging/receive.py

```diff
@@ -0,0 +1,97 @@
+from channels.db import database_sync_to_async
+from django.contrib.contenttypes.models import ContentType
+from .models import Message
+
+
+async def receive_message(text_data_json: dict) -> dict:
```

Comment:
Add a new line above this for a comment describing what it does
(e.g. A function for creating a message event)